File: terceiro_semestre/laboratorio-de-programacao/aula10/ex01/src/Book.java
public class Book {
    private String Title;
    private String Author;
    private double Price;

    public Book(String title, String author, double price) {
        this.Title = title;
        this.Author = author;
        this.Price = price;
    }

    public Book(String title, String author) {
        this.Title = title;
        this.Author = author;
        this.Price = 0.00;
    }
}

File: sexto_semestre/desenvolvimento-de-sistemas-moveis/library-in-dart/lib/models/book.dart
class Book {
  String title;
  String author;
  int publicationYear;
  String genre;
  String id;

  Book({
    required this.title,
    required this.author,
    required this.publicationYear,
    required this.genre,
    required this.id,
  });

  void displayDetails() {
    print('---');
    print('ID: $id');
    print('Title: $title');
    print('Author: $author');
    print('Publication Year: $publicationYear');
    print('Genre: $genre');
    print('---');
  }
}
